sparse_strips/vello_hybrid/src/schedule.rs

```diff
@@ -252,15 +335,32 @@ struct TileEl {
 struct Draw(Vec<GpuStrip>);
 
 impl Scheduler {
+    /// Adjust a stored round from tile state to account for flushes that occurred
+    /// after the tile started processing.
+    fn adjust_round(&self, stored_round: usize, tile_base_round: usize) -> usize {
```

Comment:
I'm not sure about this. For now, I'm just trying to get a single wide tile to work before fixing this logic.